README.md

```diff
@@ -9,19 +9,72 @@ UI Component library for developing [DHIS2](http://www.dhis2.org) applications.
 
 --
 
-### Get It! 
+## Get It!
 
 __Latest stable version for your dhis2 release__
 
-> Use @<dhis2-release-version> to get the latest version applicable to that installation
+> Use @{dhis2-release-version} to get the latest version applicable to that installation
 
-For DHIS2 2.25
+For example, if the DHIS2 release version is 2.28, then:
 ```
-npm install --save d2-ui@25
+yarn add d2-ui@28
 ```
 
-For DHIS2 2.24
+## Contributing
+
+### Development
+
+The git repository is located here: [d2-ui github repo](https://github.com/dhis2/d2-ui)
+
+Submit your change as a pull request. Pull requests need to include:
+1. good quality unit tests
+2. relevant changes/additions to the examples
+
+##### Examples
+The repo includes a webpack dev server that provides a page showing most of the d2-ui components. To run these do:
+```
+yarn start
+```
+
+Some of the examples require a local DHIS2 instance. Most developers have their DHIS2 instance running on http://localhost:8080. If this is the case for you, then the examples should just work.
+
+
+### Publishing to npm
+
+The d2-ui library is published as an npm package: https://www.npmjs.com/package/d2-ui
+
+#### A note about d2-ui semver
+The d2-ui version tracks the DHIS2 version. So, when DHIS2 version 2.28 is released, then d2-ui will be upgraded to v28.0.0. Therefore, all version upgrades prior to the next DHIS2 release are generally patch upgrades, even if the changes are considered breaking.
+
+#### Steps to publish
+
+After your pull request with functional changes has been merged to master, switch to the master branch locally and pull to get all the latest changes. The version upgrade will be done directly on the master branch:
+
 ```
-npm install --save d2-ui@24
+git co master
```

Comment:
I don't think `co` is a default binding in Git, we should be more specific in the docs and use `checkout`.

Reply:
Agree. Will fix..